GitUtils.ps1

```diff
@@ -312,13 +330,19 @@ function Start-SshAgent([switch]$Quiet) {
         if (!$pageant) { Write-Warning "Could not find Pageant."; return }
         Start-Process -NoNewWindow $pageant
     } else {
-        $sshAgent = Get-Command ssh-agent -TotalCount 1 -ErrorAction SilentlyContinue
-        $sshAgent = if ($sshAgent) {$sshAgent} else {Find-Ssh('ssh-agent')}
-        if (!$sshAgent) { Write-Warning 'Could not find ssh-agent'; return }
+        $sshService = Find-Service('ssh-agent')
+        if($sshService) {
+            Start-Service ssh-agent -ErrorAction SilentlyContinue 
+            $pid = Find-SericePID('ssh-agent')

```

Comment:
You've already got the service as `$sshService` so could avoid calling `Find-ServicePID` with `($sshService | Get-Process).Id`